README.md

```diff
@@ -2,4 +2,29 @@
 [![Stories in Progress](https://badge.waffle.io/Yolgie/jku-lan-se.png?label=in%20progress&title=In%20Progress)](https://waffle.io/Yolgie/jku-lan-se)
 # Project JKU-LAN Software
 
-Create/find software supported solutions for a few recurring problems
+## Introduction
+
+This piece of software is developed by the ÖH Informatik in order to support the organization of the JKU-LAN. 
+
+## Setup
+
+In order to run this project, you need a working installation of Gradle and JDK. Once you have this software available
+simply clone the current master branch and use "gradle bootRun" in order to run the application. 
+
+For the most popular IDE's, Eclipse and IntelliJ Idea, there is also a gradle command available in order to make all
+dependencies loaded by gradle, available. 
+
+For Eclipse: gradle eclipse
+For IntelliJ Idea: gradle idea
+
+## Work Convention
+
+To easy smooth cooperation, please follow these [guidelines](https://guides.github.com/introduction/flow/). 
```

Comment:
We are using the [Github workflow](https://guides.github.com/introduction/flow/):

1.  Keep the master clean
2.  Create a branch/pull-request for every code change. Branch name should start with the issue number, be all english lowercase with dash "-" instead of spaces
3.  Keep your commit messages to this format: `issueNumber-worktag-shortDescription`. 
  + Worktags: feature, test, refactoring, fix
```
We are using the [Github workflow](https://guides.github.com/introduction/flow/):

1.  Keep the master clean
2.  Create a branch/pull-request for every code change. Branch name should start with the issue number, be all english lowercase with dash "-" instead of spaces
3.  Keep your commit messages to this format: `issueNumber-worktag-shortDescription`. 
  + Worktags: feature, test, refactoring, fix
```